### ISSUE TITLE
not getting realtime message 

### ISSUE THREAD

**Issue by sankaran1408** [rpp0/gr-lora#177]
when i am trying to send a lora message by a transmitter and receiving real time and decoding using [lora_receive_realtime.py](http://lora_receive_realtime.py/) i am getting this
** ([lora_receive_realtime.py:130](http://lora_receive_realtime.py:130/)): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /run/user/1000/at-spi/bus: No such file or directory
libGL error: pci id for fd 7: 1002:1636, driver (null)
libGL error: No driver found
libGL error: failed to load driver: (null)
gr-osmosdr 0.1.4 (0.1.4) gnuradio 3.7.11
built-in source types: file fcd rtl rtl_tcp uhd hackrf bladerf rfspace airspy 
Using device #0 Realtek RTL2838UHIDIR SN: 00000001
Found Rafael Micro R820T tuner
Exact sample rate is: 1000000.026491 Hz
Bits (nominal) per symbol: 5.5
Bins per symbol: 2048
Samples per symbol: 16384
Decimation: 8 ....why i am not getting the decoded message ?please help
